workspace/apps/pidp/src/app/features/auth/pages/login/login.page.html

```diff
@@ -93,6 +94,18 @@ <h3>Special Authority eForms</h3>
               </p>
             </ui-page-section>
 
+            <ui-page-section>
+              <h3>Prescription Refill Request eForm</h3>
+              <p>
+                B.C. Pharmacists can enrol for access to Prescription Refill
+                Request eForm Application
+              </p>
+              <p>
+                You need to set up this BC service Card app to enrol. You will
+                user the app each time you access the eForm application.
+              </p>
+            </ui-page-section>
+
```

Comment:
The Prescription Refill eForm enrollment is still feature flagged, so we shouldn't add it to the login page yet.
We are considering other approaches in PIDP-392.

Reply:
OK I've removed it